src/Fakta/Types.fs

```diff
@@ -47,10 +47,46 @@ type Port = uint16
 
 type Check = string
 
+type CheckUpdate =
+  {
+    status : string
+    output : string
+  }
+
+  static member GetUpdateJson (st : string)(out : string) =
+    let res = {
+                  status = st;
+                  output = out
+              }
+    res
+
+  static member ToJson (chu : CheckUpdate) =
+    Json.write "Status" chu.status
+    *> Json.write "Output" chu.output
+
 type Node =
   { node    : string
     address : string }
 
+  static member emptyNode =
+    let en = {
+                node = String.Empty;
```

Comment:
Should this be an option at call-site rather than usage of `empty`? Does a Node value with an empty node property value make sense?